src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -26,11 +26,15 @@ license_header:
 collections:
   - name_pattern: shelves/{shelf_id}
     entity_name: shelf
+invalid_collections:
```

Comment:
I think I'd like to avoid the use of 'invalid' because it can be misinterpreted.